pipeline/review/main.go

```diff
@@ -76,60 +55,102 @@ func main() {
 
 // realMain executes the Commit Review Pipeline.
 func realMain(ctx context.Context) error {
-	// parse flags
+	flagGitHubToken := flag.String("github-token", "",
+		"Token to use to authenticate with github.")
+	flagGitHubAppID := flag.String("github-app-id", "",
+		"Provisioned GitHub App reference.")
+	flagGitHubAppInstallationID := flag.String("github-app-installation-id", "",
+		"Provisioned GitHub App Installation reference.")
+	flagGitHubAppPrivateKeyResourceName := flag.String("github-app-private-key-resource-name", "",
+		"Resource name for the secret manager resource containing the GitHub App private key.")
+	flagPushEventsTable := flag.String("push-events-table", "",
+		"Name of the push events table. The value provided must be a fully qualified BigQuery table name of the form <project>:<dataset>.<table>")
+	flagCommitReviewStatusTable := flag.String("commit-review-status-table", "",
+		"Name of the commit review status table. The value provided must be a fully qualified BigQuery table name of the form <project>:<dataset>.<table>")
+	flagIssuesTable := flag.String("issues-table", "",
+		"name of the issues table. The value provided must be a fully qualified BigQuery table name of the form <project>:<dataset>.<table>")
```

Comment:
Curious - does it make a difference if we define these vars outside of the `readMain` without the `init()`. Like

```go
var (
  flagGitHubToken := flag.String("github-token", "",
		"Token to use to authenticate with github.")
  // ...
)

func readMain() {}
```

It seems more common to define flags in a `var` group

Reply:
In general, it'll be slower. Those allocations take place during build and during boot time, whereas moving them into the function means they won't be allocated until absolutely needed. This is more important for CLIs (where load time is super critical), but it applies here as well.